crates/solidity/outputs/npm/tests/src/parser/parse.test.mts

```diff
@@ -0,0 +1,60 @@
+import { Parser } from "@nomicfoundation/slang/parser";
+import { NonterminalKind, TerminalKind, assertNonterminalNode, assertTerminalNode } from "@nomicfoundation/slang/cst";
+import { LanguageFacts } from "@nomicfoundation/slang/utils";
+
+function new_parser(): Parser {
+  return Parser.create(LanguageFacts.latestVersion());
```

Comment:
I suggest inlining this expression for clarity.
If not, using camelCase for JavaScript.